tests/test_timeout.py

```diff
@@ -10,7 +10,7 @@
 class FakeAsyncStream(asyncio.StreamReader):
     async def read(self, n: int) -> bytes:
         await asyncio.sleep(delay=2)
-        return bytes([0] * n)
+        raise NotImplementedError("This line should not be reached")
```

Comment:
It'd be nice if there was something explaining why this line shouldn't be reached.
```suggestion
        raise NotImplementedError(
            "This override of read method isn't intended for actual use!\n"
            " - If you're writing a new test, did you forget to mock it?\n"
            " - If you're seeing this in an existing test, this method got called without the test expecting it,"
            " this probably means you changed something in the code leading to this call, but you haven't updated"
            " the tests to mock this function."
        )
```

Reply:
Fixed in [last force-push](https://github.com/py-mine/mcstatus/compare/62efe7dd58b02749d77abb934e7caa04d7ff66ed..a2dcf432d2b93004bff7d1f55f952025ec38f4e9).

Reply:
`asyncio.sleep` needs here for testing the `timeout` option. If remove this call, function will immediately raise exception, and will not be killed by timeout handler.